profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module

```diff
@@ -10,6 +10,13 @@ include_once 'nexteuropa_metatags.tokens.inc';
  * Implements hook_metatag_info().
  */
 function nexteuropa_metatags_metatag_info() {
+  $info['group']['tags'] = array(
```

Comment:
I had to add this at some point when we had the issue with the metatags, since errors were generated.